### PR TITLE
Text Input

### PR DESCRIPTION
**Pull request by sarahyaepark**
Updated text area component to work for numbers and shorter inputs as well as multiline.
- added valid checker functionality, but it only checks that it is not empty for now
inputarea_demo

### CODE REVIEW

File: react/client/public/index.html [hunk 1]
```diff
@@ -48,6 +48,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
```
Comment (aahvocado) [inline react/client/public/index.html:51]: Could you use icons via the npm package instead? https://www.npmjs.com/package/@material-ui/icons

File: react/client/src/components/Textarea.tsx [hunk 2]
```diff
@@ -28,27 +37,75 @@ interface TextFieldProps {
...
   placeholder,
+  multi,
 }) => {
+  const checkValid = (e: string) => {
+    isValid(e.length > 0);
+  };
+  const [valid, isValid] = useState(false);
   const classes = useStyles();
-  return (
+  return inputName === 'age' ? (
```
Comment (aahvocado) [inline react/client/src/components/Textarea.tsx:54]: Haha `Input` and `Textarea` are two different things - I would like to see them as separate components. 

On the web the function very differently with one being a single line and the other be multiline. There's also implications for the html itself, such as there being the `value` attribute on an input element.

Comment (sarahyaepark) [reply react/client/src/components/Textarea.tsx:54]: Okay! Updating now.

File: react/client/public/index.html (hunk 1 above)
Comment (sarahyaepark) [reply react/client/public/index.html:51]: Got it!

**Review by aahvocado — APPROVED** [review]

**Pull request merged**